charts/v4.6.0/csi-driver-nfs/templates/csi-nfs-controller.yaml

```diff
@@ -98,7 +98,7 @@ spec:
           securityContext:
             readOnlyRootFilesystem: true
         - name: nfs
-          image: "{{ .Values.image.nfs.repository }}:{{ .Values.image.nfs.tag }}"
+          image: "{{ .Values.image.repourl }}{{ .Values.image.nfs.repository }}:{{ .Values.image.nfs.tag }}"
```

Comment:
this may break existing chart users since only nfs.repository is set in old versions, what about write like following?

```
        - name: nfs
{{- if hasPrefix "/" .Values.image.nfs.repository }}
          image: "{{ .Values.image.baseRepo }}{{ .Values.image.nfs.repository }}:{{ .Values.image.nfs.tag }}"
{{- else }}
          image: "{{ .Values.image.nfs.repository }}:{{ .Values.image.nfs.tag }}"
{{- end }}
```

Reply:
example: https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/fda1d0a3174ce628d57ce01a86b15a492a77f583/charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml#L152C1-L157C11